include/ddc/kernels/splines/spline_builder.hpp

```diff
@@ -17,7 +17,9 @@ namespace ddc {
  * An enum determining the backend solver of a SplineBuilder or SplineBuilder2d. Only GINKGO available at the moment,
  * other solvers will be implemented in the futur.
  */
-enum class SplineSolver { GINKGO };
+enum class SplineSolver {
+  GINKGO ///< Enum member to identify the Ginkgo-based solver (iterative method)
+};
 
 /**
  * @brief An helper giving the uniform/non_uniform status of a spline interpolation mesh according to its attributes.
```

Comment:
```suggestion
 * @brief A helper giving the uniform/non_uniform status of a spline interpolation mesh according to its attributes.
```

Reply:
Same 2 lines below. "An" should only be used before a word starting with a vowel